tendrl/notifier/notification/__init__.py

```diff
@@ -118,6 +118,12 @@ def run(self):
                     NS.config.data["notification_check_interval"]
                 )
                 time.sleep(interval)
+                lock = etcd.Lock(NS._int.client, 'alerting')
+                lock.acquire(blocking=True,
+                             lock_ttl=60)
+                if lock.is_acquired:
+                    # renew a lock
+                    lock.acquire(lock_ttl=60)
```

Comment:
Why need renewal of lock?

Reply:
Let's take example node A and Node B are the two machines which will try to push the alerts. If node A acquires the lock and proceeding further, So at same time node B also trying but it unable to acquire lock so it has to wait for some time. When node A finishes then node B acquired lock but it already lost some time on waiting. So renewing the lock will help to give another 60 sec to finish the node B job.

Reply:
Ack. Changes look fine